iosViewBuilder/iosViewBuilder/GlobalProperties.swift

```diff
@@ -33,17 +33,20 @@
 import SwiftUI
 import SharedMobileUI
 
-let bodyFontSize: CGFloat = 18
-let titleFontSize: CGFloat = 32
-let stepIconFontSize: CGFloat = 16
+let bodyFontSize: CGFloat = 20
```

Comment:
FYI: This file is in the wrong place and won't get built into the MotorControlUI library. We can make a task to sort that out when cleaning up the package